spec/lib/active_versioning/model/versioned_spec.rb

```diff
@@ -0,0 +1,122 @@
+require 'spec_helper'
+
+RSpec.describe ActiveVersioning::Model::Versioned do
+  let!(:user) { ActiveVersioning::Test::User.create(name: 'Bo Janglez', email: 'bo.janglez@gmail.com') }
+
+  subject { ActiveVersioning::Test::Post.create(title: 'So Post', body: 'Such interesting.  Very wow.', user: user) }
+
+  it { should have_many(:versions) }
+
+  describe "after create" do
+    it "creates a new version" do
+      expect {
+        ActiveVersioning::Test::Post.create(title: 'So Post', body: 'Such interesting.  Very wow.', user: user, version_author: user)
```

Comment:
you could use `build` to instantiate the subject and just call `subject.save` here if you want to clean things up